CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [5.0.0]
```

Comment:
For the breaking change:
```
 **BREAKING:** Remove  `metamaskNetworkId` from smart transaction state ([#191](https://github.com/MetaMask/smart-transactions-controller/pull/191))
 ```
 we should maybe add a sub bullet indicating what, if anything, consumers need to do to gracefully absorb this change

Reply:
done: https://github.com/MetaMask/smart-transactions-controller/pull/202/commits/6953d8c50a300413baaaf701859ef84bbe743c1f